index.html

```diff
@@ -171,7 +169,7 @@ <h2 class="footer-title">I'm always interested in hearing about new projects, so
             </li>
             <li class="submit-button">
               <button type="submit">Get in touch</button><br />
-              <small></small>
+              <div class="error_msg"></div>
```

Comment:
Consider moving this line/element before the submit button. This is because keeping the error message after the submit button can somethings make it harder for the user to notice that something changed on the screen.